legal-api/migrations/versions/ec9652171563_review.py

```diff
@@ -0,0 +1,56 @@
+"""review
+
+Revision ID: ec9652171563
+Revises: 01b28a2bb730
+Create Date: 2024-06-26 16:13:07.413995
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'ec9652171563'
+down_revision = '01b28a2bb730'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('filings', sa.Column('resubmission_date', sa.DateTime(timezone=True), nullable=True))
+
+    op.create_table('reviews',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('nr_number', sa.String(length=15), nullable=True),
+                    sa.Column('identifier', sa.String(length=50), nullable=True),
+                    sa.Column('completing_party', sa.String(length=150), nullable=True),
+                    sa.Column('status', sa.Enum('AWAITING_REVIEW', 'CHANGE_REQUESTED', 'RESUBMITTED', 'APPROVED', 'REJECTED', name='review_status'), nullable=False),
+                    sa.Column('submission_date', sa.DateTime(timezone=True), nullable=True),
+                    sa.Column('creation_date', sa.DateTime(timezone=True), nullable=True),
+                    sa.Column('filing_id', sa.Integer(), nullable=False),
+                    sa.ForeignKeyConstraint(['filing_id'], ['filings.id'], ),
+                    sa.PrimaryKeyConstraint('id')
+                    )
+    op.create_table('review_results',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('status', sa.Enum('AWAITING_REVIEW', 'CHANGE_REQUESTED', 'RESUBMITTED', 'APPROVED', 'REJECTED', name='review_status'), nullable=False),
+                    sa.Column('comments', sa.Text(), nullable=True),
+                    sa.Column('reviewer_id', sa.Integer(), nullable=True),
+                    sa.Column('creation_date', sa.DateTime(timezone=True), nullable=True),
+                    sa.Column('submission_date', sa.DateTime(timezone=True), nullable=True),
+                    sa.Column('review_id', sa.Integer(), nullable=False),
+                    sa.ForeignKeyConstraint(['review_id'], ['reviews.id'], ),
+                    sa.ForeignKeyConstraint(['reviewer_id'], ['users.id'], ),
+                    sa.PrimaryKeyConstraint('id')
+                    )
```

Comment:
What would a review result record look like before the staff enter any comments or change the status -- in other words, with only the user submission recorded?

Reply:
Do you have a sample response for the "get review" endpoint (ticket bcgov/entity#21985)?

Reply:
there won't be a review_result until staff review (approved/change request/rejected)

Reply:
How then will we show the user resubmission?

![image](https://github.com/bcgov/lear/assets/10002889/a4cdcdd9-6392-490f-928b-97475b2c3d00)

Reply:
at this point there is no resubmission (only user submission). if review_result endpoint returns empty then show the creation_date from review endpoint as user submission

Reply:
I was hoping that the "get review" endpoint would return everything needed:
- review object
- review results array (possibly empty)

Will there be 1 review per continuation application, and 1 or more review results?

Reply:
you are right. it will be one api call

Reply:
```
review = {

review_results=[] - this array contains 0 value initially and 1 or more entry based on the staff review (Correspondence)
}
```